ansible/group_vars/searchengine_vars.yml

```diff
@@ -0,0 +1,13 @@
+apps_folder: /data
+database_server_url: "{{ omero_db_host_ansible }}"
```

Comment:
When executing the playbook, this leads to `'omero_db_host_ansible' is undefined`. I think the metadata will need to be extracted using a `set_fact` task similar to https://github.com/IDR/deployment/blob/db10eafc4c7f131ff14d4cd639a59a94a2751478/ansible/idr-omero.yml#L27-L32